src/DocumentWatcher.ts

```diff
@@ -177,7 +177,7 @@ function applyEditorConfigToTextEditor(
 	);
 
 	/* tslint:disable:no-any */
-	textEditor.options = <any> newOptions;
+	textEditor.options = newOptions as any;
```

Comment:
Yeah. I read about it recently, because it's the only syntax that works in a TSX file for React in some scenarios.